app/talk/comment-image-selector.cjsx

```diff
@@ -19,6 +20,7 @@ module?.exports = React.createClass
       .then (subjects) =>
         subject.type = 'recent' for subject in subjects
         @setState {subjects}
+      .catch =>
```

Comment:
Since the button is waiting for the promise to be resolved, if a subject isn't found it never resolves.